src/applications/gi/containers/LandingPage.jsx

```diff
@@ -1,6 +1,15 @@
+/* eslint-disable prettier/prettier */
 import React from 'react';
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router';
+import * as LDClient from 'launchdarkly-js-client-sdk';
+import { withLDProvider, withLDConsumer } from 'launchdarkly-react-client-sdk';
+
+const user = {
```

Comment:
- users are required part of the feature flag analytics 
- You can set up user reports to be anonymized. 
- You can send any arbitrary information about a user and use that to segment the features e.g. you could capture a user's email address and have feature flag rules that puts that user on a beta test set of features based on that email address.